_includes/pbc/all/install-features/202304.0/install-warehouse-picking-feature.md

```diff
@@ -0,0 +1,534 @@
+This document describes how to integrate the Warehouse picking feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Warehouse picking feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME                                    | VERSION          | INTEGRATION GUIDE                                                                                                                                                                  |
+|-----------------------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Warehouse User Management               | {{site.version}} | [Install the Warehouse User Management feature](/docs/scos/dev/feature-integration-guides/{{site.version}}/install-the-warehouse-user-management-feature.html)                     |
+| Order Management + Inventory Management | {{site.version}} | [Order Management and Inventory Management feature](/docs/scos/dev/feature-integration-guides/{{site.version}}/install-the-order-management-and-inventory-management-feature.html) |
+| Shipment                                | {{site.version}} | [Install the Shipment feature](/docs/scos/dev/feature-integration-guides/{{site.version}}/shipment-feature-integration.html)                                                       |
+| Picker User Login                       | {{site.version}} |                                                                                                                                                                                    |
```

Comment:
What is it? Is it a future stand-alone feature? I can't find it in the ReleaseGroup app.

Reply:
This feature right now in development.